### DISCUSSION TITLE
6장 - purgatory(퓨거토리)에 대해서 깊게 알아두면 좋을까?

### DISCUSSION THREAD

**Discussion by kyungjunleeme** [kyungjunleeme/kafka_study#63] [Q&A]
- purgatory
- https://www.confluent.io/blog/apache-kafka-purgatory-hierarchical-timing-wheels/
- https://dol9.tistory.com/282
- https://github.com/kafkakru/meetup/blob/master/meetup/2022-meetup/2nd-virtual-meetup/Purgatory%EB%8A%94%20%EB%AC%B4%EC%97%87%EC%9D%B8%EA%B0%80%3F.pdf

**Comment by sejongjeong** [comment]
https://dol9.tistory.com/m/281

https://www.confluent.io/blog/apache-kafka-purgatory-hierarchical-timing-wheels/

https://ferbncode.github.io/Apache-Kafka-and-Request-Purgatory.html

**Comment by kyungjunleeme** [comment]
chatgpt따르면 큐는 아니라고 하는데 자세히 살펴보자
